FinniversKit.podspec

```diff
@@ -25,7 +25,7 @@ Pod::Spec.new do |s|
   s.author             = "FINN.no"
   s.social_media_url   = "https://twitter.com/search?q=FINN.no"
 
-  s.platform     = :ios
+  s.platform = :ios, '9.0'
```

Comment:
Do we need the `, 9.0` part for the platform? Since the deployment_target is listed below.

Reply:
Removed 👍